src/bin/package_for_release/main.rs

```diff
@@ -6,10 +8,26 @@ mod utils;
 
 #[cfg(target_os = "linux")]
 fn main() {
-    bundle_linux::bundle_linux_app().unwrap();
+    env_logger::init_from_env(
+        env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+    );
+
```

Comment:
leave this file as it was, putting this logic here only complicates thinking what happens where, but you are still duplicating code, it's better to keep it in bundle_linux/bundle_macos